src/cljs/main/broadfcui/auth.cljs

```diff
@@ -159,6 +159,34 @@
                                        :redirectUri (.. js/window -location -origin)})
                                :on-done #(swap! state assoc :hidden? true)})))))})
 
+(defn force-signed-in [{:keys [on-sign-in on-sign-out on-error]}]
+  (fn [auth-token]
+    (utils/ajax {:url (str "https://www.googleapis.com/oauth2/v3/tokeninfo?access_token=" auth-token)
```

Comment:
Do you think there is any need for `(js/encodeURIComponent auth-token)` here?

Reply:
Good suggestion. I assume it shouldn't hurt for valid input and anything we can do to sanitize input is a good thing. I'll try it to make sure it doesn't have any bad side effects.